src/ui/templates/insurance/page-not-found.njk

```diff
@@ -0,0 +1,27 @@
+{% extends 'index.njk' %}
+
+{% block pageTitle %}
+  {{ CONTENT_STRINGS.PAGE_TITLE }}
+{% endblock %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-l" data-cy="heading">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
+
+  <p class="govuk-body" data-cy="body-1">{{ CONTENT_STRINGS.BODY_1 }}</p>
```

Comment:
I wonder if it's better making the name more descriptive then BODY_1 and BODY_2? not a blocker

Reply:
Great shout, renamed to `TYPED_ADDRESS` and `PASTED_ADDRESS`